code_doc_gen/config.py

```
@@ -318,13 +356,60 @@ def get_ai_config(self) -> Dict[str, Any]:
+                'mistral_api_key': 'mistral'
+            }
+            
+            for config_key, provider_name in providers_map.items():
+                if not env_keys.get(config_key):
+                    stored_key = cred_manager.get_key(provider_name)
+                    if stored_key:
+                        env_keys[config_key] = stored_key
+                        
+        except ImportError:
+            pass
+        except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)